pt-BR/active_record_validations.md

```diff
@@ -833,18 +833,18 @@ Topic.create(title: nil).valid? # => true
 
 ### `:message`
 
-As you've already seen, the `:message` option lets you specify the message that
-will be added to the `errors` collection when validation fails. When this
-option is not used, Active Record will use the respective default error message
-for each validation helper. The `:message` option accepts a `String` or `Proc`.
+Como você já viu, a opção `:message` permite que você especifique a mensagem que
+será adicionada à coleção `errors` quando a validação falhar. Quando está
+opção não é usada, o `Active Record` usará a respectiva mensagem de erro padrão
+para cada `helper` de validação. A opção `:message` aceita uma `String` ou `Proc`.
```

Comment:
```suggestion
para cada *helper* de validação. A opção `:message` aceita uma `String` ou `Proc`.
```

Acho que podemos deixar em itálico pra marcar mas sem marcar o código, eu também acho que prefiro código. Mas é um padrão das outras páginas 😞